packages/cli/src/cli/basemaps/tileset.updater.ts

```diff
@@ -126,6 +144,21 @@ export async function updateConfig(filename: string, tag: TileMetadataTag, isCom
     }
 }
 
+function extractSetConfig(a: TileMetadataSetRecord | ProjectionConfig): TileMetadataSetConfig {
+    const ans: any = {};
```

Comment:
Might be better to have this as a `Partial<TileMetadataSetConfig>`

Reply:
That doesn't appear to work:

```
Type 'string | RGBAColor | { in: TileResizeKernel; out: TileResizeKernel; } | { in: string; out: string; }' is not assignable to type '(string & RGBAColor & { in: TileResizeKernel; out: TileResizeKernel; }) | undefined'.
  Type 'string' is not assignable to type '(string & RGBAColor & { in: TileResizeKernel; out: TileResizeKernel; }) | undefined'. [2322]
```